test/ToucanRegenBridge.js

```diff
@@ -2,26 +2,30 @@
 const { expect } = require("chai");
 const { ethers } = require("hardhat");
 
-// import { solidity } from "ethereum-waffle";
-// chai.use(solidity);
+const { deployBridge } = require("../lib/bridge");
+const { prepareToucanEnv } = require("../lib/toucan");
 
 describe("Token contract", function () {
```

Comment:
can we fix indentation in another PR?

Reply:
Fine by me. It seems we are missing linting config?